.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+##### Description
+
+<!---
+Please include a short description of what your PR does and / or the motivation behind it
+--->
+
+##### Checklist
+
+- [ ] Built with `cargo build`
+- [ ] Formatted with `cargo fmt`
+<!-- - [ ] Linted with `cargo clippy` -->
```

Comment:
We don't pass clippy ourselves, yet, so asking contributors to run clippy will give them warnings and errors from code they didn't touch.